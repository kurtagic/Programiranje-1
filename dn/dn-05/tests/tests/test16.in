7
1 95
1 72
1 76
0 66
2 78
2 75
0 99
