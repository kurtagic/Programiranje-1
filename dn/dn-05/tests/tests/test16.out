66: 1
67: 1
68: 1
69: 1
72: 4
75: 6
76: 6
78: 2
79: 2
95: 4
99: 1
100: 1
101: 1
102: 1
